src/systems/ec/implementations/vanilla/packages/ec-component-form-checkbox/ec-component-form-checkbox.scss

```diff
@@ -0,0 +1,104 @@
+/**
+ * Checkbox component
+ * @define checkbox
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@include check-imports(('ec-component-icon'));
+
+@mixin ecl-checkbox(
+  $thickness-base: 1px,
+  $thickness-max: 3px,
+  $border-color: $ecl-color-grey-50,
+  $border-color-focus: $ecl-color-yellow-100,
+  $border-color-checked: $ecl-color-blue-100,
+  $fake-checkbox-size: 0.875rem
+) {
+  .ecl-checkbox {
+    align-items: center;
+    display: flex;
+    margin: 0;
+    position: relative;
+  }
+
+  .ecl-checkbox__input {
+    clip: rect(1px, 1px, 1px, 1px);
+    height: 1px;
+    overflow: hidden;
+    position: absolute !important;
+    width: 1px;
+  }
+
+  .ecl-checkbox__label {
+    color: $ecl-color-grey-100;
+    font: $ecl-font-m;
+  }
+
+  .ecl-checkbox__label--hidden {
+    clip: auto;
+    height: auto;
+    margin: 0;
+    position: static;
+    text-indent: -9999px;
+    width: auto;
+  }
+
+  .ecl-checkbox__box {
+    border: $thickness-base solid $border-color;
+    display: inline-flex;
+    height: $fake-checkbox-size;
```

Comment:
Ok, I have to ask; why didn't you use the width/height from the specs? (`1.25rem`)
It would have been much easier... Just use `1.25rem` with a `box-sizing: border-box`, remove the padding (why is there a padding anyway?), and leverage the `inline-flex` with `justify-content: center; align-items: center;`. This way, the checkbox will be use the values from the specs and the icon will be automatically centered.

Reply:
well, in a default content-box implementation the padding was compensating the different border-size and computing the requested size, but i agree, it's simply better using border-box.